pyproject.toml

```diff
@@ -23,6 +23,9 @@ sqlalchemy-utils = "~0.41.1"
 uvicorn = "~0.23.2"
 fastapi-pagination = "^0.12.12"
 fastapi-filter = "^1.0.0"
+pytest = "^7.4.3"
```

Comment:
All dev dependencies must be added to the "dev" group (below): `poetry add --group=dev pytest`
And pytest is already included :)

Reply:
oops, you're right